amd/src/iconpicker.js

```diff
@@ -92,8 +102,8 @@ function searchicon(event) {
  * Adds class "highlight" to currently selected icon.
  */
 function highlightselected() {
-    let icons = Array.from(document.getElementsByClassName('block_floatingbutton-iconpicker-icon'));
-    let input = document.querySelector('.block_floatingbutton-iconpicker').getAttribute('data-iconpicker-input');
+    let icons = Array.from(document.getElementsByClassName(SELECTORS.iconpicker_icon));
```

Comment:
after extracting the selector as selector, this has to be `document.querySelectorAll` instead of `document.getElementsByClassName`. Otherwise the hightlighting of a currently selected icon does not work anymore.